projects/wave-core-new/src/lib/datasets/upload/upload.component.ts

```diff
@@ -438,20 +439,38 @@ export class UploadComponent {
         return undefined;
     }
 
-    private getColumnsAsMap(): {[key: string]: 'categorical' | 'int' | 'float' | 'text'} {
+    private getColumnsAsMap(): {[key: string]: VectorColumnInfoDict} {
         const formMeta = this.formMetaData.controls;
-        const columns: {[key: string]: 'categorical' | 'int' | 'float' | 'text'} = {};
+        const columns: {[key: string]: VectorColumnInfoDict} = {};
 
         for (const column of formMeta.columnsText.value as Array<string>) {
-            columns[column] = 'text';
+            columns[column] = {
+                dataType: 'text',
+                measurement: {
+                    // TODO: incorporate in selection
```

Comment:
should we create a card for that or is it not important?

Reply:
Which dialog allows creating a `Measurement` other than unitless?
I guess no one, not for any operator or this upload.
So you say, is this important? I guess it could be a RESPECT task?

Reply:
I am not sure. But if we say: "ok measurements are now supported in Geo Engine", but you can't specify them for your own data, I guess at some point we will have a problem. But we can also just wait for that moment instead of creating a card that no one will work on anyway. Fine for me 😀

Reply:
Well, the changelog is for the backend. And they were there for rasters a long time ago.
I hesitate to make UI changes that are not useful for something :shrug:.